docs/pages/versions/v48.0.0/sdk/location.mdx

```diff
@@ -63,17 +63,24 @@ You can configure `expo-location` using its built-in [config plugin](/guides/con
       default: '"Allow $(PRODUCT_NAME) to use your location"',
     },
     {
-      name: 'isIosBackgroundLocationEnabled',
+      name: 'locationWhenInUsePermission',
       platform: 'ios',
       description:
         'A string to set the [`NSLocationWhenInUseUsageDescription`](#permission-nslocationwheninuseusagedescription) permission message.',
       default: '"Allow $(PRODUCT_NAME) to use your location"',
     },
+    {
+      name: 'isIosBackgroundLocationEnabled',
+      platform: 'ios',
+      description:
+        'A boolean to enable `location` in the `UIBackgroundModes` in **Info.plist**.',
+      default: '',
+    },
     {
       name: 'isAndroidBackgroundLocationEnabled',
       platform: 'android',
       description:
-        'A string to set the [`ACCESS_BACKGROUND_LOCATION`](#permission-access_background_location) permission message.',
+        'A boolean to enable the [`ACCESS_BACKGROUND_LOCATION`](#permission-access_background_location) permission.',
       default: '',
```

Comment:
This defaults to `false` (the code uses `if (isAndroidBackgroundLocationEnabled)`) so not setting it is effectively equal to `false`. Booleans without a default (tri-state booleans) trigger me :)